### PR TITLE
Salesforce enhancement

### PR DESCRIPTION
**Pull request by tzemel**
## Status
In Progress, 
The test unit is needed to complete the pull request.

Salesforce needs to be able to retrieve comments from a specific case number.
Salesforce integration needs to be able to post a comment from a specific case number.
Salesforce integration needs to ability to get user name from a specific case number.
Salesforce integration needs to ability to get organization from a specific case number.

Command Name:
- salesforce-get-casecomment
- salesforce-post-casecomment
- salesforce-get-user 
- salesforce-get-org

## Related Issues
fixes: https://github.com/demisto/etc/issues/20963

## Description
PANW Demisto cs team needs to add a few new commands for salesforce integration.

## Screenshots
SalesForce

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](Integrations/integration-Salesforce.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/Salesforce_Enhancement/Integrations/integration-Salesforce_CHANGELOG.md)
[Api documentation](https://developer.salesforce.com/docs/api-explorer/sobject/CaseComment)

### CODE REVIEW

**Comment by tzemel** [comment]
Added Ymal File

**Comment by richardbluestone** [comment]
Reviewed and updated documentation

File: Integrations/integration-Salesforce.yml [hunk 1]
```diff
@@ -332,12 +443,27 @@ script:
...
+            entries.push(orgToEntry(cases, 'Account:', userMapping));
+        }
+
         if (case_ids.length > 0) {
             condition = "ID IN ('" + case_ids.join("','") + "')";
-            properties = ['ID', 'CaseNumber', 'Subject', 'Description', 'CreatedDate', 'ClosedDate', 'OwnerID', 'Priority', 'Origin', 'Status', 'Reason'];
-            var cases = queryObjects(properties, "Case", condition).records;
-            entries.push(casesToEntry(cases, 'Cases:', userMapping));
+            properties = ['ID', 'CaseNumber', 'Subject', 'Description', 'CreatedDate', 'ClosedDate', 'OwnerID', 'Priority', 'Origin', 'Status', 'Reason','IsEscalated','SuppliedPhone','SuppliedCompany','SuppliedEmail','ContactEmail','ContactId','AccountId'];
+            var casesA = queryObjects(properties, "Case", condition).records;
```
Comment (IkaDemisto) [inline Integrations/integration-Salesforce.yml]: No need to rename `cases` variable.

**Comment by IkaDemisto** [comment]
@tzemel you have reverted all the descriptions changes that @richardbluestone pushed, from this commit 0bfaa1b56a1d31676ec68e8cfd40a7fbdbed4cea. Please make sure to fix it, before we merge this pr.

**Comment by IkaDemisto** [comment]
@tzemel approved

**Pull request merged**